lang/java/pom.xml

```diff
@@ -38,12 +38,7 @@
 
     <!-- version properties for dependencies -->
 
-    <!--
-      To build the avro-mapred module against Hadoop 1 specify
-      -Dhadoop.version=1 or leave unspecified to build against Hadoop 2
-    -->
-    <hadoop1.version>1.2.1</hadoop1.version>
-    <hadoop2.version>2.7.3</hadoop2.version>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
with the coming release of Hadoop 3, is it worth future proofing ourselves by keeping {{hadoop2.version}} to ease the future addition of a {{hadoop3.version}} ?